lib/flowViewport/sceneIndex/fvpDefaultMaterialSceneIndex.cpp

```diff
@@ -22,13 +22,33 @@
 #include <pxr/imaging/hd/materialBindingsSchema.h>
 #include <pxr/imaging/hd/containerDataSourceEditor.h>
 #include <pxr/imaging/hd/retainedDataSource.h>
+#include <pxr/usd/usdGeom/tokens.h>
 
 namespace FVP_NS_DEF {
 
 PXR_NAMESPACE_USING_DIRECTIVE
 
 namespace{
     static const TfToken purposes[] = { HdMaterialBindingsSchemaTokens->allPurpose };
+
+    // Support implicit surfaces from USD as well, not only meshes
+    bool _IsDefaultMaterialCompliantPrimitive(const TfToken& primType) 
+    { 
+        static std::set<TfToken> const compliantPrimitives = { HdPrimTypeTokens->cone,
+                                                              HdPrimTypeTokens->cylinder,
+                                                              HdPrimTypeTokens->cylinder_1,
+                                                              HdPrimTypeTokens->cube,
+                                                              HdPrimTypeTokens->sphere,   
+                                                              HdPrimTypeTokens->capsule,
+                                                              HdPrimTypeTokens->capsule_1,
+                                                        #if HD_API_VERSION >= 67 // USD 24.05+
+                                                              UsdGeomTokens->TetMesh,
+                                                        #endif
+                                                              UsdGeomTokens->Plane,
+                                                              HdPrimTypeTokens->mesh};
+        return std::find(compliantPrimitives.cbegin(), compliantPrimitives.cend(), primType)
+            != compliantPrimitives.cend();
```

Comment:
Must use std::set::find() here to get better performance, and it also makes the code more compact. I also think std::set::count() is even more compact, but that's personal preference and std::set::find() is fine
```suggestion
        return compliantPrimitives.count(primType) == 1;
```